include/gz/physics/Joint.hh

```diff
@@ -584,6 +586,44 @@ namespace gz
       };
     };
 
+    /////////////////////////////////////////////////
+    /// \brief This feature sets the mimic constraint for this Joint.
+    class GZ_PHYSICS_VISIBLE SetMimicConstraintFeature
+        : public virtual Feature
+    {
+      /// \brief The Joint API for setting the mimic joint constraint.
+      /// \param[in] _mimicJoint
+      ///   name of the joint to be mimicked, i.e. the parent joint.
+      /// \param[in] _multiplier
+      ///   The multiplier to be applied to poistion of parent joint.
+      /// \param[in] _offset
+      ///   The offset to be applied to position of parent joint after
+      ///   the multiplier is applied.
+      public: template <typename PolicyT, typename FeaturesT>
+      class Joint : public virtual Feature::Joint<PolicyT, FeaturesT>
+      {
+        public: using Scalar = typename PolicyT::Scalar;
+
+        public: void SetMimicConstraint(
+                    const std::string _mimicJoint,
```

Comment:
let's add the axis name string here to match the SDFormat API

Reply:
Done https://github.com/gazebosim/gz-physics/pull/431/commits/6a0294b1c790dd7950843d6aa75acf52bcf7c054